tests/test_forms.py

```diff
@@ -192,58 +173,53 @@ def test_form_changed_when_only_one_value_is_changed():
     assert bool(form.changed_data) == True
 
 
-def test_render_money_input():
-    widget = widgets.MoneyInput(choices=[("USD", "US dollar")], attrs={"key": "value"})
-    result = widget.render("price", Money(5, "USD"), attrs={"foo": "bar"})
-    assert 'foo="bar"' in result
-    assert 'name="price_0"' in result
-    assert 'name="price_1"' in result
-    assert 'key="value"' in result
-    assert 'value="5"' in result
-    assert "USD" in result
-
-
-def test_render_fixed_currency_money_input():
-    widget = widgets.FixedCurrencyMoneyInput(currency="BTC", attrs={"key": "value"})
-    result = widget.render("price", Money(5, "BTC"), attrs={"foo": "bar"})
-    attrs = ['foo="bar"', 'key="value"', 'value="5"', "price_0", "BTC"]
-    for attr in attrs:
-        assert attr in result
+def test_form_field_passes_all_validations_for_correct_money_value():
+    form = RequiredPriceForm(data={"price_net_0": "20", "price_net_1": "BTC"})
+    form.full_clean()
+    assert form.errors == {}
 
 
-def test_money_field_instance_init_by_money_object():
-    instance = Model(price_net=Money(25, "USD"))
-    assert instance.price.net.amount == 25
+def test_form_field_fixed_currency_andrequired_pass_on_valid_data():
+    form = FixedCurrencyRequiredPriceForm(data={"price_0": "20", "price_1": "BTC"})
+    form.full_clean()
+    assert form.errors == {}
+    assert form.cleaned_data["price"] == Money(20, currency="BTC")
 
 
-def test_taxed_money_field_instance_gross_and_net_are_set_separately():
-    instance = Model(price_net=Money(25, "BTC"), price_gross=Money(30, "BTC"))
-    assert instance.price == TaxedMoney(net=Money(25, "BTC"), gross=Money(30, "BTC"))
+def test_form_field_fixed_currency_and_required_error_on_wrong_currency():
+    form = FixedCurrencyRequiredPriceForm(data={"price_0": "20", "price_1": "LOL"})
+    form.full_clean()
+    assert form.errors == {
+        "price": ["Select a valid choice. LOL is not one of the available choices."]
+    }
 
 
-def test_money_field_instance_full_clean_raises_error_for_invalid_amount(db):
-    model = Model(price_gross=Money("10.999", "BTC"))
-    with pytest.raises(ValidationError):
-        model.full_clean()
+def test_form_field_fixed_currency_and_required_error_on_missing_amount():
+    form = FixedCurrencyRequiredPriceForm(data={"price_1": "BTC"})
+    form.full_clean()
+    assert form.errors == {"price": ["This field is required."]}
 
 
-def test_taxed_money_field_set_instance_value():
-    instance = Model()
-    instance.price = TaxedMoney(Money(25, "BTC"), Money(30, "BTC"))
-    assert instance.price_net == Money(25, "BTC")
-    assert instance.price_gross == Money(30, "BTC")
+def test_form_field_fixed_currency_and_optional_pass_on_valid_data():
+    form = FixedCurrencyOptionalPriceForm(data={"price_0": "20", "price_1": "BTC"})
+    form.full_clean()
+    assert form.errors == {}
+    assert form.cleaned_data["price"] == Money(20, currency="BTC")
```

Comment:
You don't need to explicitly pass currency as a keyword argument. `Money` only takes two arguments.